app/v1/models/response/retrieveBsas/selfEmployment/Metadata.scala

```diff
@@ -37,7 +37,7 @@ object Metadata {
     (JsPath \ "inputs" \ "incomeSourceType").read[IncomeSourceType].map(_.toTypeOfBusiness) and
       (JsPath \ "inputs" \ "incomeSourceId").readNullable[String] and
       JsPath.read[AccountingPeriod] and
-      (JsPath \ "metadata" \ "taxYear").read[String].map(DesTaxYear.fromDes) and
+      (JsPath \ "metadata" \ "taxYear").read[Int].map(DesTaxYear.fromDesIntToString) and
```

Comment:
Can't we just use the old function and write something like `.read[Int].map(DesTaxYear.fromDes(_.toString))`?